### DISCUSSION TITLE
Keeping track about various efforts that use SSSOM online - add you own if you like!

### DISCUSSION THREAD

**Discussion by matentzn** [mapping-commons/sssom#318] [Show and tell]
Lets use this discussion to keep track of various places online that implement SSSOM one way or another. This can later be user to develop something like as showcase for documentation.

**Comment by matentzn** [comment]
https://github.com/jmillanacosta/mappings_enm

**Comment by matentzn** [comment]
https://github.com/brain-bican/metadata-schemas/tree/dev/docs/mappings

**Comment by matentzn** [comment]
https://github.com/British-Oceanographic-Data-Centre/NvsVocPrez

**Comment by matentzn** [comment]
https://github.com/MIT-LCP/mimic-code/tree/main/mimic-iv/concepts/concept_map (seems to be only reusing some elements)

**Comment by matentzn** [comment]
https://github.com/clnsmth/soso/tree/development/src/soso/data

**Comment by matentzn** [comment]
https://github.com/microbiomedata/nmdc-schema/tree/main/sssom

**Comment by matentzn** [comment]
https://github.com/qudt/qudt-public-repo/tree/main/community/mappings/SSSOM/IFC

**Comment by matentzn** [comment]
https://github.com/jhu-bids/TermHub

**Comment by matentzn** [comment]
https://github.com/tdwg/gbwg/tree/main/dwc-mixs/mapping

**Comment by matentzn** [comment]
https://github.com/geneontology/pathways2GO/tree/master/exchange/src/main/resources/YeastCyc

**Comment by matentzn** [comment]
https://docs.google.com/presentation/d/1jB1zRZvv0E_g6BPN4lpC5K3hAQt_wfziLb4TdfrlFPE/edit#slide=id.g285a545345d_2_0

**Comment by matentzn** [comment]
https://github.com/monarch-initiative/koza/pull/109/files

**Comment by matentzn** [comment]
https://github.com/dwslab/melt/blob/master/yet-another-alignment-api/src/main/java/de/uni_mannheim/informatik/dws/melt/yet_another_alignment_api/DefaultExtensions.java#L825

**Comment by matentzn** [comment]
https://github.com/Planteome/CO_337-groundnut-traits/issues/7

**Comment by matentzn** [comment]
https://github.com/bridge2ai/clive

**Comment by matentzn** [comment]
https://github.com/bge-barcoding/StayingMapped

**Comment by matentzn** [comment]
https://github.com/berkeleybop/artificial-intelligence-ontology/tree/main/src/mappings

**Comment by matentzn** [comment]
https://github.com/tdwg/mids/tree/main/sssom-mapping

**Reply by matentzn** [reply]
Maybe worth supporting them getting the TSV representation right; its a bit idiosyncratic, but does not have to be!

**Comment by matentzn** [comment]
https://f1000research.com/articles/13-1102

**Comment by matentzn** [comment]
https://www.nature.com/articles/s41597-024-04168-1

**Comment by matentzn** [comment]
https://www.ohdsi.org/wp-content/uploads/2024/10/25-houghtaling-endtoendontology-2024global-report-Jared-Houghtaling.pdf

**Comment by matentzn** [comment]
https://github.com/cns-iu/onto-llm-mapping/

**Comment by matentzn** [comment]
https://github.com/BFO-Mappings/PROV-to-BFO/blob/main/prov-bfo-directmappings.ttl

**Comment by matentzn** [comment]
https://github.com/LiUSemWeb/Circular-Economy-Ontology-Catalogue/blob/main/alignments/task_a/CAMO-CEO/CAMO-CEO-sssom-AMD.ttl

**Comment by gouttegd** [comment]
Mappings between EFO and a few other ontologies, from RGD:

https://download.rgd.mcw.edu/ontology/mappings/

(produced by an ad-hoc “SSSOM File Generator” whose code is here: https://github.com/rat-genome-database/ontology-load-pipeline/blob/master/src/main/java/edu/mcw/rgd/dataload/ontologies/SSSOMGenerator.java)

**Comment by matentzn** [comment]
https://health-ri.github.io/semantic-interoperability/ontology/mappings/

**Comment by matentzn** [comment]
https://github.com/IBM/risk-atlas-nexus/

**Comment by matentzn** [comment]
https://biss.pensoft.net/article/181877/

**Reply by matentzn** [reply]
Related: https://biss.pensoft.net/article/181898

**Comment by matentzn** [comment]
https://mapping.bio/

**Comment by matentzn** [comment]
https://davidchavesfraga.com/outcomes/papers/2026/code2026llm.pdf

## LLM Driven Justified Ontology Alignment

Ontology alignment is a key task for achieving semantic interoperability across heterogeneous knowledge graphs;
yet, it remains a time-consuming and expert-driven process. Recent advances in Large Language Models (LLMs)
offer new opportunities to automate this task, particularly in scenarios involving the parallel development of
multiple ontologies by automatic means that follow different approaches. In this paper, we propose an approach
for the automatic generation of ontology alignments using LLMs, producing mappings that are compliant with
the SSSOM standard and enriched with explicit justifications and provenance metadata that can support ontology
developers and domain experts in developing ontologies from multiple automatically generated versions. We
design and evaluate a set of progressively refined prompts to guide LLMs in generating structured and explainable
alignments. The approach is assessed using multiple state-of-the-art models (GPT-5.4, GPT-5 Mini, Gemini Flash,
and Gemini Pro) against a selection of ontology pairings from the OAEI Conference dataset. The evaluation
combines structural validation, standard alignment metrics (precision, recall, and F1-score), and expert qualitative
analysis. The results observed indicate that LLMs possess the potential to generate high-quality candidate
mappings, particularly for lexically similar entities. However, limitations persist in semantic discrimination,
predicate selection, and the exploitation of ontology structure. These findings indicate that LLMs are best suited
as assistive tools for knowledge engineers and domain experts in managing the parallel evolution of ontologies.

**Comment by matentzn** [comment]
https://github.com/EVORA-project/virus-taxonomy-mappings/tree/dev/mappings

**Comment by matentzn** [comment]
SSSOM cited by an ISO standard: https://www.iso.org/obp/ui/en/#iso:std:iso:21564:ed-1:v1:en

I dont know how its cited as the ISO standard is closed but, its cited.

**Comment by matentzn** [comment]
https://github.com/National-Clinical-Cohort-Collaborative/Data-Ingestion-and-Harmonization/blob/master/sssom_mapping_commons/example/sssom_mappings_1678icd10cm.csv

**Comment by matentzn** [comment]
https://github.com/KE-UniLiv/animl-ontology/blob/main/alignments/outputs/alignments.csv

**Reply by matentzn** [reply]
https://arxiv.org/html/2604.01728v1

Files dont seem valid (missing mapping_justification)

**Reply by gouttegd** [reply]
Also uses invalid format for most ID-typed fields (either full-length IRIs such as `http://purl.allotrope.org/ontologies/common#AFC_0000162` or single keywords such as `partOf` or `LogMap`, instead of proper CURIEs).

And even if the file was actually valid, the scarcity of metadata in it makes this sentence of the paper looks like a joke:

> All alignments are released using the Simple Standard for Sharing Ontological Mappings (SSSOM) [43], **a format that provides relevant metadata to describe each mapping.**

Yeah, that’s certainly what the SSSOM format _allows_ to do… then why don’t they do it? :roll_eyes:

**Comment by jmillanacosta** [comment]
https://github.com/jmillanacosta/pysec2pri (will be moved to https://github.com/sec2pri/pysec2pri soon) collects deprecated/consolidated/split entry identifiers from databases and generates release-specific mapping sets. It also collects and qualifies (when possible) alias/synonym/etc labels.

**Comment by matentzn** [comment]
A schema mapping use case implemented in SSSOM. Bigger use of extensions slots in the wild: https://github.com/daakiv/ohdsi-gis-metadata-template/blob/main/authoring/kobotoolbox/mappings/sssom/kobo_testing_merged.sssom.tsv

I am getting to know that community and might suggest that there could be better ways to represent the transforms; but nice.

**Comment by matentzn** [comment]
That is a very idiosyncratic but interesting use: https://github.com/OO-LD/oold-schema/issues/107

**Comment by cthoyt** [comment]
Nico posted this on OBO slack last week:

> New AIES 2026 paper on why AI risk taxonomies keep getting built but rarely used (ANU/Cornell/USC/Microsoft Research/Google Research) — their fix for the mess includes publishing cross-taxonomy mappings in SSSOM, so we’re being named as infrastructure for a field well outside biomedicine: https://arxiv.org/abs/2608.06831

Referring to ["Death by a thousand taxonomies?": AI Risk Classification In Practice](https://arxiv.org/abs/2608.06831) by [Berman](https://arxiv.org/search/cs?searchtype=author&query=Berman,+G) et al, 2026

**Reply by gouttegd** [reply]
> their fix for the mess includes publishing cross-taxonomy mappings in SSSOM, so we’re being named as infrastructure

Err, all they say in this paper is:

> this mapping can be supported by adopting stable semantic identifiers for risks, and using frameworks **like** the Simple Standard for Sharing Ontological Mappings.

(The emphasis on “like” is mine.) From there to “SSSOM being named as infrastructure”, I think that’s stretching it a little bit too far.

Now, they refer to another paper (which I’m pretty sure has been mentioned already, if not here, then probably on the OBO Slack): [Bagehorn et al. (2025)](https://arxiv.org/pdf/2503.05780), whose authors do more than merely _mentioning_ SSSOM – they effectively use it (https://github.com/IBM/ai-atlas-nexus/tree/main/src/ai_atlas_nexus/data/mappings).

Though not always correctly, looking at some of their rows:

| subject_id | subject_label | predicate_id | object_id | object_label | mapping_justification | author_id | confidence |
|------------|---------------|--------------|-----------|--------------|-----------------------|---------|------------|
| ibm-risk-atlas:atlas-toxic-output | toxic output | noMatch | owasp-llm-2.0:llm052025-improper-output-handling | LLM05:2025 Improper Output Handling | semapv:ManualMappingCuration | EMAIL_REDACTED | 0.95 |

The use of `noMatch` as a mapping predicate is invalid. This seems to be their way of representing “negative mappings”?

Also, the use of a naked email address (which I have redacted here) as `author_id` is invalid as well, because an email address is not a CURIE.

**Comment by matentzn** [comment]
https://pmc.ncbi.nlm.nih.gov/articles/PMC13577442/

**Reply by cthoyt** [reply]
I looked through the paper and the repository they linked and didn't find any evidence they actually used SSSOM :/

what they wrote was:

> We curated SSSOM-compatible metadata for 142,113 mapping relationships [[21](https://pmc.ncbi.nlm.nih.gov/articles/PMC13577442/#R21)]. Mapping precision has 3 categories: broadMatch (mapping with loss of granularity, 17% of the annotated mappings), narrowMatch (mapping with increased granularity, <1%), and exactMatch (83%). Metadata for concepts were also curated and distributed, including categories of nonstandard codes, as described in the Vocabularies’ interoperability documentation, and a reuse flag identifying codes reused across source terminologies [[18](https://pmc.ncbi.nlm.nih.gov/articles/PMC13577442/#R18)]. Concept and relationship metadata were made publicly available and accompany each release [[22](https://pmc.ncbi.nlm.nih.gov/articles/PMC13577442/#R22)].

**Reply by matentzn** [reply]
I did find some evidence but its deeply buried. Use is a bit of an overstatement I would say. They took various SSSOM metadata elements and added them straight into their own model: the OMOP vocabulary model: https://github.com/search?q=repo%3AOHDSI%2FVocabulary-v5.0+sssom&type=wikis

Also, they added a few SSSOM attributes to their community contribution templates.

In any case - this is not a wholesale adoption, for sure.
